src/Column/BigBitColumn.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Pgsql\Column;
+
+use Yiisoft\Db\Constant\ColumnType;
+use Yiisoft\Db\Expression\ExpressionInterface;
+use Yiisoft\Db\Schema\Column\AbstractColumn;
+
+use function decbin;
+use function gettype;
+use function str_pad;
+
+use const STR_PAD_LEFT;
+
+final class BigBitColumn extends AbstractColumn
+{
+    protected const DEFAULT_TYPE = ColumnType::BIT;
+
+    /** @psalm-suppress RedundantCast */
+    public function dbTypecast(mixed $value): string|ExpressionInterface|null
```

Comment:
May be extract this code to separate place to prevent logic duplication with `BitColumn::dbTypecast()`?